ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/route.tsx

```diff
@@ -74,10 +74,12 @@ const Layout = () => {
   ];
 
   return (
-    <div className='flex h-full w-full gap-2'>
+    <>
       <Sidebar sections={sections} />
-      <Outlet />
-    </div>
+      <div className='w-full md:max-w-4xl'>
+        <Outlet />
+      </div>
+    </>
```

Comment:
This sort of pattern is not repated in many `route.tsx` files. It could make sense to extract it to a component. [Here's](https://primer.style/components/page-layout/react/alpha) an example of what the API could look like.

Reply:
Okay, now I've added a suggestion of one way of doing this